packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuoteFilter.tsx

```diff
@@ -0,0 +1,62 @@
+import { Select } from '@trezor/components';
+import styled from 'styled-components';
+import { useTranslation } from 'src/hooks/suite';
+import {
+    PaymentMethodListProps,
+    UseCoinmarketFilterReducerOutputProps,
+} from 'src/hooks/wallet/useCoinmarketFilterReducer';
+import { CoinmarketPaymentPlainType } from 'src/views/wallet/coinmarket/common/CoinmarketPaymentPlainType';
+
+const Wrapper = styled.div`
+    margin-top: 20px;
```

Comment:
Use spacings if there is such value
```suggestion
    margin-top: ${spacingsPx.lg};
```